metagraph/tests/graph/all/test_dbg_helpers.cpp

```diff
@@ -39,6 +40,9 @@ template<> size_t max_test_k<DBGHashFast>() {
 template<> size_t max_test_k<DBGHashString>() {
     return 100;
 }
+template<> size_t max_test_k<DBGSSHash>() {
+    return 255 / kmer::KmerExtractor2Bit::bits_per_char;
```

Comment:
`DBGSSHash` currently uses the value from `IDictionary` rather than from `KmerExtractor2Bit`.